modules/ROOT/partials/menu-item-ids/advtable-menu-items.adoc

```diff
@@ -1,11 +1,15 @@
+[cols="d,d,a"]
```

Comment:
Likewise here, I'm not sure if we want this?

Reply:
Done